File: apps/desktop/src/5_entities/instances/model/installStage.ts
export enum InstanceInstallStage {
  /// Instance is installed
  Installed = 'installed',
  /// Instance's minecraft game is still installing
  Installing = 'installing',
  /// Instance created for pack, but the pack hasn't been fully installed yet
  PackInstalling = 'pack_installing',
  /// Instance is not installed
  NotInstalled = 'not_installed',
}

File: packages/core/aether-core/src/features/instance/domain/model/install_stage.rs
#[derive(Clone, Copy, Debug, Eq, PartialEq)]
pub enum InstanceInstallStage {
    /// Instance is installed
    Installed,
    /// Instance's minecraft game is still installing
    Installing,
    /// Instance created for pack, but the pack hasn't been fully installed yet
    PackInstalling,
    /// Instance is not installed
    NotInstalled,
}
